apps/codeforafrica/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,48 @@
+import { Section } from "@commons-ui/core";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import NewsletterSubscription from "@/codeforafrica/components/NewsletterSubscription";
+
+const FooterRoot = styled(Box)(
+  ({ theme: { breakpoints, palette, typography } }) => ({
+    backgroundColor: palette.common.black,
+    color: palette.text.secondary,
+    padding: `${typography.pxToRem(80)} 0`,
+    [breakpoints.up("md")]: {
+      padding: `${typography.pxToRem(110)} 0`,
+    },
+    [breakpoints.up("lg")]: {
+      padding: `${typography.pxToRem(100)} 0`,
+    },
+  })
+);
+
+const Footer = React.forwardRef(function Footer(props, ref) {
+  const { subscription } = props;
+
+  return (
+    <FooterRoot ref={ref}>
+      <Section>
+        <Grid container justifyContent="flex-end">
```

Comment:
I am guessing `flex-end` is just a solution for current situation (1 item at the far end).. cause it may confuse the next developer adding component to this `Grid container`.
 
Why not define the other two grid as empty 
`<Grid item xs={12} md={6} />`
`<Grid item xs={12} md={3} />`

Reply:
Yeah, I thought about it but .. **[YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it)**. I don't want to limit the next dev options, they can choose to continue with the `Grid` (and add missing `Grid item`s) or even completely switch to a different container/structure.